openapi/components/schemas/QuoteChangeOrder.yaml

```
@@ -227,6 +227,12 @@ properties:
       This field is only applicable if `renewalPolicy` is set to `retain`.
     type: boolean
     default: false
+  interimOnly:
+    type: boolean
+    default: true
+    description: |-
+      Specifies if quotation invoice must include interim items only.
+      If set to `false` all upcoming items are included.
```

suggestion: for readability 
```suggestion
      If this value is set to `false`, all upcoming items are included.
```